ballerina/googlesheets_client.bal

```diff
@@ -259,7 +256,9 @@ public isolated client class GoogleSheetsClient {
                 } else if (key is int|string|decimal|float) {
                     values.push(key);
                 }
-            } else if !updateRecord.hasKey(entityKey) && self.keyFields.indexOf(entityKey) == () {
+            }
+
+            else if !updateRecord.hasKey(entityKey) && self.keyFields.indexOf(entityKey) == () {
```

Comment:
```suggestion
            } else if !updateRecord.hasKey(entityKey) && self.keyFields.indexOf(entityKey) == () {
```

Reply:
Fixed in [6af274a](https://github.com/ballerina-platform/module-ballerinax-persist.googlesheets/pull/15/commits/6af274ac3b3c8e6595db3232d142746015cd4ef0)